app/frontend/src/components/NotificationBar/_NotificationBar.scss

```diff
@@ -0,0 +1,97 @@
+// External Imports
+@use "sass:map";
+
+// Internal Imports
+@use "../Basics" as *;
+
+/*****************************
+*** Notification Bar class ***
+*****************************/
+$themes: "primary", "success", "warning", "danger";
+
+.notification-bar {
+  height: 55px;
+}
+
+.notification-bar-text {
+  flex: 1;
+}
+
+.show {
+  visibility: visible;
+}
+
+.hide {
+  visibility: hidden;
+}
```

Comment:
We have [utility functions](https://github.com/hackforla/CivicTechJobs/blob/3c4ea0be150e02a25b6370e3a4d531026f79b78a/app/frontend/src/components/Basics/_utils.scss#L3) that can serve the function of these classes. Let me know if they are viable for your use cases.

There's also probably no need for both a show and hide, since if a class is removed, React will just remake the dom element, which resets everything to its default visibility.